### PR TITLE
Redirect to webmail signatures settings

### PR DESCRIPTION
**Pull request by tevincent**
https://github.com/Infomaniak/android-kMail/assets/149579879/6871edf7-2b90-4fd1-a25a-455430d9a87e

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -300,6 +300,7 @@
         <item quantity="one">Indirizzo e-mail bloccato</item>
         <item quantity="other">Indirizzi e-mail bloccati</item>
     </plurals>
+    <string name="manageSignatures">Gestire le mie firme</string>
```
Comment (LunarX) [inline app/src/main/res/values-it/strings.xml:303]: Aks @valentinperignon about the conjugation of that sentence

File: app/src/main/res/layout/fragment_signature_setting.xml [hunk 2]
```diff
@@ -45,6 +45,14 @@
                 app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
                 tools:itemCount="3"
                 tools:listitem="@layout/item_settings_signature" />
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/manageSignatures"
+                style="@style/TextButtonSecondary"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/manageSignatures" />
```
Comment (LunarX) [inline app/src/main/res/layout/fragment_signature_setting.xml:54]: Either make the padding of the button so the actual text of the button is aligned horizontally with the list of signatures' actual text or make it so the ripple of the button is horizontally aligned with the start of the signatures' actual text but not something in between

File: app/build.gradle [hunk 3]
```diff
@@ -37,6 +37,7 @@ android {
         // buildConfigField 'String', 'MAIL_API_PREPROD', '"https://mail.preprod.dev.infomaniak.ch"'
         buildConfigField 'String', 'SHOP_URL', '"https://ik.me"'
         buildConfigField 'String', 'CHATBOT_URL', '"https://www.infomaniak.com/chatbot"'
+        buildConfigField 'String', 'MANAGE_SIGNATURES_URL', '"https://mail.infomaniak.com/0/settings/signatures"'
```
Comment (LunarX) [inline app/build.gradle:40]: Do we create a whole variable for it? Or do we create an ApiRoute that depends on `MAIL_API`?

**Review by LunarX — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 3 above)
Comment (tevincent) [reply app/build.gradle:40]: @Infomaniak/android what do you all think?

Comment (KevinBoulongne) [reply app/build.gradle:40]: We already have `'MAIL_API', '"https://mail.infomaniak.com"'`.
Re-use it, like the other routes are doing.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 3 above)
Comment (KevinBoulongne) [reply app/build.gradle:40]: Keep it this way.

**Review by KevinBoulongne — APPROVED** [review]

File: app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (tevincent) [reply app/src/main/res/values-it/strings.xml]: @valentinperignon sent me the new string.

**Review by LunarX — APPROVED** [review]

**Pull request merged**